cg/constants/orderforms.py

```diff
@@ -31,9 +31,8 @@ class Orderform(StrEnum):
 
 
 ORDERFORM_VERSIONS = {
```

Comment:
Could we make this a classmethod for Orderform and not a detached constant?

Reply:
Made it a `static_method`